tests/40-sd-create.js

```diff
@@ -38,7 +38,7 @@ describe('ecdsa-sd-2023 (create)', function() {
           this.implemented.push(`${name}: ${keyType}`);
           describe(`${name}: ${keyType}`, function() {
             // find matching verifier
-            const verifier = implementation.verifiers.filter(
+            const verifier = implementation.verifiers.find(
```

Comment:
find returns the first verifier that meets the qualifications of the predicate function. filter returns an array. the problem is the verifier was not being popped out of the array resulting in an error that was not being reported because verificationSucess was not awaited. So find returns an actual verifier wrapped in an `Endpoint` es6 Class. This has a few wins: 1. the `should.exist` check for verifier will actually throw if a verifier is not found (an empty array from filter will probably pass `should.exist`) 2. we don't have destructure the array to get the verifier.